src/test/scala/org/camunda/feel/interpreter/impl/builtin/BuiltinNumberFunctionTest.scala

```diff
@@ -39,6 +39,17 @@ class BuiltinNumberFunctionsTest
     eval(" decimal(2.5, 0) ") should be(ValNumber(2))
   }
 
+  "A decimal() function with rounding mode" should "return number with a given scale" in {
```

Comment:
We can group all tests for one function together.

```suggestion
  it should "round the number using the given rounding mode" in {
```